.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,59 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fluent UI Apple? This template includes necessary information to get started, and steps to completion
+---
+
+<!-- Use this template for new components or new component variants -->
+
+## Component details
+
+### Description
+
+<!-- fill this out -->
+
+### Which product teams/scenarios need this control
+
+<!-- The more teams/scenarios that would use this control the better chance it will get prioritized -->
+
+### Design assets
+
+<!-- Please provide links to redlines or screenshots of intended component design -->
+
+### Component ownership
+
+<!-- Are there one or more people who can help maintain this component over time? Who will address bugs? -->
+
+## Component composition
+
+### Imports/Dependencies
+
+<!-- What other dependencies will your component be using -->
+
+### Proposed Public API
+
+<!-- Define the proposed public API for this control -->
+
+### Intended Platforms
+- [ ] iOS
+- [ ] macOS
+
+## Component code
+
+### Example code
+
+```Swift
+
+```
+
+### New Component Checklist
+
+- [ ] Light and Dark Appearance
+- [ ] Unit tests
+- [ ] UI tests
```

Comment:
possible iPad / iPhone check?

Reply:
I'll use the same wording I do in the pull request template, whatever we decide on.